content/blog/2021-03-31-march-21-community-gems.md

```diff
@@ -0,0 +1,128 @@
+tags:
+  - Discord
+  - Gems
+  - Pipelines
+  - Remote Storage
+---
+
+### [Q: Will DVC work with <my remote cloud storage of choice?>](https://discord.com/channels/485586884165107732/563406153334128681/821493606770409493)
+
+We recently had questions about this, specifically regarding Huawei Cloud and
+Backblaze B2 Storage. The answer is any cloud storage that has an S3 interface
+will work with DVC and both of the aforementioned do! In addition DVC works with
+Azure, Google Drive, GS, OSS and SSH.
+[Learn more about S3 combatibility integrations and all available remote storage capabilities here.](https://dvc.org/doc/command-reference/remote)
+
+Thanks to @luke and @Samuel H from Discord for asking these questions that led
+to this Gem! 💎
+
+### [Q: I had understood previously that DVC was not suitable for hyperparameter tuning. Has that changed?](https://discord.com/channels/485586884165107732/485596304961962003/820722752709328967)
+
+Yes indeed! With DVC 2.0, the capabilities have evolved quite a bit! We have
+introduced experiments and metrics which enables you to track and compare the
+different runs of your models with various hyperparameters. You can check out
+the documents [here](https://dvc.org/doc/start/experiments) and
+[here](https://dvc.org/doc/start/metrics-parameters-plots) to see all the
+details.
+
+Thanks to @saif3r for helping us highlight the new features in DVC!
+
+### [Q: Is it possible to set up a DVC repo with pipelines which have all the data (cache, input, output) on another (local) location outside the repo?](https://discord.com/channels/485586884165107732/485596304961962003/819509440217874473)
+
+Thanks for the question @EEisbrenner!
+
+One solution to this would be to keep your DVC cache on your mount, and use the
+`symlink` cache type so all of your data would remain on that mount, but for
+DVC's purposes it would only deal with files that are "inside" your repo (via
+symlinks). Note that your data on that mount would be stored in DVC's
+content-addressable cache format, and not in `path/to/mount/foo.nc`. Check out
+the docs on
+[how to keep DVC cache on your mount here.](https://dvc.org/doc/use-cases/shared-development-server)
+
+To actually work with `foo.nc`, you'd end up with a symlink `foo.nc` inside your
+git/DVC repo that points to some object in your DVC cache.  
+[See these docs](https://dvc.org/doc/user-guide/large-dataset-optimization) for
+info on how the cache link types work. For doing the initial `dvc add` step for
+your data without needing to copy it into the DVC/repo first,
+[check out these docs](https://dvc.org/doc/command-reference/add#example-transfer-to-the-cache).
+
+### [Q: My peers and I share a repo where we have a folder that is versioned with DVC. I'm getting an error message when trying to pull data from the cloud. What could be causing it?](https://discord.com/channels/485586884165107732/563406153334128681/799617584336338954)
+
+I see you are having the following error:
+
+```bash
```

Comment:
bash -> dvc

start each command with `$`:

```
$ dvc pull
```

Reply:
+ add some space between `dvc pull` and `dvc doctor` please

Reply:
ok.  I think I get it now...